pkg/operators/v1alpha1/catalogsource_types.go

```diff
@@ -242,6 +243,28 @@ func (c *CatalogSource) Poll() bool {
 	return true
 }
 
+// UnmarshalJSON implements the encoding/json.Unmarshaler interface.
+func (u *UpdateStrategy) UnmarshalJSON(data []byte) (err error) {
+	var updateStrategyMap map[string]map[string]string
+	if err = json.Unmarshal(data, &updateStrategyMap); err != nil {
+		return err
+	}
+	registryPoll := &RegistryPoll{}
+	duration, err := time.ParseDuration(updateStrategyMap["registryPoll"]["interval"])
+	if err != nil {
+		err = nil
+		defaultTime, err := time.ParseDuration("15m")
+		if err != nil {
+			return err
+		}
```

Comment:
nit: 
```suggestion
	if err != nil {
		defaultTime, parseErr := time.ParseDuration("15m")
		if parseErr != nil {
		    return parseErr
		}
```

Reply:
Also, the `15m` default time should be initialized as a constant and the constant should be used here instead of hard coding the value here.